src/Tribe/Repositories/Attendee/RSVP.php

```diff
@@ -77,4 +112,109 @@ public function checked_in_keys() {
 		return $this->limit_list( $this->key_name, parent::checked_in_keys() );
 	}
 
+	/**
+	 * Filter the arguments to set for the attendee for this provider.
+	 *
+	 * @since TBD
+	 *
+	 * @param array                         $args          List of arguments to set for the attendee.
+	 * @param array                         $attendee_data List of additional attendee data.
+	 * @param Tribe__Tickets__Ticket_Object $ticket        The ticket object or null if not relying on it.
+	 *
+	 * @return array List of arguments to set for the attendee.
+	 */
+	public function filter_set_attendee_args( $args, $attendee_data, $ticket = null ) {
+		// Set default order ID.
+		if ( empty( $args['order_id'] ) ) {
+			// Use a regular variable so we can call a static method from it PHP <7.0
```

Comment:
:no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: e7ded55ef35e99134abd7802ce1a44ea</sub>